app/src/main/java/org/oppia/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -1021,6 +1125,7 @@ class StatePlayerRecyclerViewAssembler private constructor(
     fun build(): StatePlayerRecyclerViewAssembler {
       val playerFeatureSet = featureSets.reduce(PlayerFeatureSet::union)
       return StatePlayerRecyclerViewAssembler(
+        adapterBuilder.build(),
         adapterBuilder.build(),
```

Comment:
Please rename one of these adapterBuidler

Reply:
added inline comment instead because it is only one adapterBuilder